### PR TITLE
[0.x] Add Fixers for Laravel specific PHPDocs

### LINKED ISSUE #1: Incorrect PHPDoc Formatting
- Pint Version: 0.1.0
- PHP Version: 8.1.6

### Description:
The Laravel Code Style has the following style throughout the source code:

#### `laravel_phpdoc_alignment`
After `@param` should be two spaces then a `type-hint` then another two spaces and then the `$variable`
```diff
  /**
   * Here is a description for what the function does
-  * @param string $foo
+  * @param  string  $foo
   * @return string
   */
  public function bar($foo) {}
```

#### `laravel_phpdoc_order`
First should be the `description` after that `@param` then `@return` and at last `@throws`
The default rule `phpdoc_order` has a different order: `description`, `@param`, `@throws`, `@return`
```diff
  /**
   * Here is a description for what the function does
   * @param string $foo
   * @param  string  $foo
-  * @throws \Exception
   * @return string
+  * @throws \Exception
   */
  public function bar($foo) {}
```

#### `laravel_phpdoc_seperation `
Between each group of tags should be an empty line for seperation. `@param` and `@return` are always of the same group and are not to be seperated with an empty line.
The default behaviour for `phpdoc_seperation` is also an empty line between `@param` and `@return`.
```diff
  /**
   * Here is a description for what the function does
+  *
   * @param string $foo
   * @param  string  $foo
-  *
   * @return string
+  *
   * @throws \Exception
   */
  public function bar($foo) {}
```

### Steps To Reproduce:
1. Create Fixtures for the example
2. Run pint
3. Check the result remains unchanged

### Possible Solution
Since there are no default rules for PHP-CS-Fixer I created custom ones.
There are three Fixers for PHP-CS-Fixer which could be included here.
https://github.com/Jubeki/laravel-code-style/tree/main/src/Fixers

Do you have an interest in a PR?

### PR DESCRIPTION
**Pull request by Jubeki**
Fixes #1 

The Laravel Code Style has the following style throughout the source code, which currently can not be described with PHP-CS-Fixer alone. This PR adds three new custom fixers for the laravel preset, which of course can be added to psr12 or symfony if the user wants to.

The Fixers are mostly copies with changes from the PHP-CS-Fixer Source Code.

### `Laravel/laravel_phpdoc_alignment`
After `@param` should be two spaces then a `type-hint` then another two spaces and then the `$variable`
```diff
  /**
   * Here is a description for what the function does
-  * @param string $foo
+  * @param  string  $foo
   * @return string
   */
  public function bar($foo) {}
```

#### `Laravel/laravel_phpdoc_order`
First should be the `description` after that `@param` then `@return` and at last `@throws`
The default rule `phpdoc_order` has a different order: `description`, `@param`, `@throws`, `@return`
```diff
  /**
   * Here is a description for what the function does
   * @param string $foo
   * @param  string  $foo
-  * @throws \Exception
   * @return string
+  * @throws \Exception
   */
  public function bar($foo) {}
```

#### `Laravel/laravel_phpdoc_seperation `
Between each group of tags should be an empty line for seperation. `@param` and `@return` are always of the same group and are not to be seperated with an empty line.
The default behaviour for `phpdoc_seperation` is also an empty line between `@param` and `@return`.
```diff
  /**
   * Here is a description for what the function does
+  *
   * @param string $foo
   * @param  string  $foo
-  *
   * @return string
+  *
   * @throws \Exception
   */
  public function bar($foo) {}
```

## Additional Note:
There is still some Fixers which need to be implemented, e.g. that in the PHPDoc is full quantified class name is used, which also cannot be described by PHP-CS-Fixer alone:
You may take a look at https://github.com/adamwojs/php-cs-fixer-phpdoc-force-fqcn
though the complex type formatting is not tagged yet and there are still some problems with classes which are in the same namespace.

### CODE REVIEW

File: app/Fixers/Utils/PhpdocTagComperator.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Fixers\Utils;
+
+use PhpCsFixer\DocBlock\Tag;
+
+final class PhpdocTagComperator
```
Comment (wilsenhc) [inline app/Fixers/Utils/PhpdocTagComperator.php:7]: Should be named `PhpdocTagComparator`

Comment (Jubeki) [reply app/Fixers/Utils/PhpdocTagComperator.php]: Thanks, fixed that.

**Comment by nunomaduro** [comment]
Can you test fix the test suite, and in general adjust the coding style of the pull request? No final classes, no types, etc.

**Comment by Jubeki** [comment]
I am sorry, but the types are defined in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/FixerInterface.php and need to be added.

I of course can remove them in `PhpdocTagComparator`.

**Comment by Jubeki** [comment]
I will remove the types from the protected and private methods which are not from the interface.

**Pull request merged**